lab-ron/route/note-route.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const storage = require('../lib/storage.js');
+const response = require('../lib/response.js');
+const Note = require('../model/note.js');
+
+
+module.exports = function(router){
+
+  router.get('/api/note', function(req, res) {
+    console.log('inside of the router function, note-route.js');
+    if (req.url.query.id) {
+      storage.fetchItem('note', req.url.query.id)
+      .then(note => {
+        res.writeHead(200, {'Content-Type': 'application/json'});
+        res.write(JSON.stringify(note));
+        res.end();
+      })
+      .catch( err => {
+        console.error(err);
+        res.writeHead(404, {'Content-Type':'text/plain'});
+        res.write('note found line 24\n');
+        res.end();
+      });
+      return;
+    } else {
+      if(!req.url.query.id) {
+        storage.fetchAll('note')
+        .then( note => {
+          res.writeHead(200, {'Content-Type': 'application/json'});
```

Comment:
Similar to the note in the fetchAll method... you're just returning the file names, and should be displaying the contents of those files.